### DISCUSSION TITLE
待翻译页面……

### DISCUSSION THREAD

**Discussion by maboloshi** [maboloshi/github-chinese#62] [Ideas]
有谁认领回复一下:
- [x] https://skills.github.com/
- [x] https://github.com/sponsors/accounts
- [x] https://github.com/sponsors/maboloshi/waitlist
- [x] https://github.com/sponsors/O0maboloshi0O/waitlist
- [ ] https://github.com/orgs/O0maboloshi0O/sponsoring
- [ ] https://github.com/orgs/O0maboloshi0O/sponsoring/billing_options
- [ ] https://github.com/orgs/O0maboloshi0O/sponsoring/invoiced_agreement_signatures/new
- [x] https://github.com/organizations/O0maboloshi0O/settings/roles
- [x] https://github.com/account/choose?action=upgrade
- [x] https://github.com/maboloshi/github-chinese/community/code-of-conduct/new
- [ ] https://github.com/organizations/O0maboloshi0O/billing/plans
- [x] https://github.com/account/upgrade?plan=pro&source=account+compare+plans
- [x] https://github.com/account/upgrade?plan_duration=month&new_plan=developer
- [ ] https://github.com/organizations/O0maboloshi0O/settings/custom-properties
- [ ] https://github.com/organizations/O0maboloshi0O/settings/custom-property
- [ ] ~~https://github.com/security~~
- [x] GitHub 任意页面底下的`管理 Cookies`和`请勿分享我的个人信息`
- [ ] https://github.com/settings/connections/applications/xxxxxx
- [ ] https://github.com/pulls (打开功能预览中的“新版拉取请求仪表板”后）

**Comment by pecasha** [comment]
[https://github.com/Rich-Harris?tab=sponsoring](https://github.com/Rich-Harris?tab=sponsoring)

**Reply by maboloshi** [reply]
已适配

**Comment by TC999** [comment]
给<https://skills.github.com/>做下正则匹配，我来试下

**Reply by maboloshi** [reply]
已适配 [439bf92](https://github.com/maboloshi/github-chinese/commit/439bf923c1938fc0bf873ba037abcffa415dc8cb)

**Reply by maboloshi** [reply]
你用新分支进行翻译吧

**Reply by TC999** [reply]
> 已适配 [439bf92](https://github.com/maboloshi/github-chinese/commit/439bf923c1938fc0bf873ba037abcffa415dc8cb)

#168

**Comment by TC999** [comment]
<https://skills.github.com/>全部翻完
#169

**Comment by TC999** [comment]
<https://github.com/about>有没有必要翻译

**Reply by maboloshi** [reply]
没有, 不常用

**Comment by TC999** [comment]
这是 Github 支持页
<https://support.github.com/>
自带中文，但不完全
真不愧是巨硬

**Reply by TC999** [reply]
试着翻译了一部分
#189

**Reply by maboloshi** [reply]
目前不打算收录，这块工作还是让官方来完成吧

**Reply by maboloshi** [reply]
还可以使用类似这种工具翻译`https://github.com/immersive-translate/immersive-translate/`

**Comment by TC999** [comment]
GitHub 官方 Git 教程<https://github.com/git-guides>

**Comment by TC999** [comment]
这个有没有必要翻译，Github 咨询页，查看软件包漏洞的网页
<https://github.com/advisories/>

**Reply by maboloshi** [reply]
使用率蛮低的，只有企业级的用户会关心

**Comment by TC999** [comment]
Github Copilot 页
<https://github.com/copilot>

**Comment by TC999** [comment]
github pages 页，搭建github.io博客的教学网站
<https://pages.github.com>

**Reply by maboloshi** [reply]
实际官网 help 有中文了 https://docs.github.com/zh/pages

**Comment by maboloshi** [comment]
@TC999 你这边的这个页面 https://github.com/settings/copilot 能截个图么 貌似我这里的有很大不同

![image](https://github.com/user-attachments/assets/84c33ac7-5b1a-4d71-b88b-f1f2502ab030)

> PS:截图扩展 FireShot

**Reply by TC999** [reply]
GitHub 把词条改了，我弄下词条

**Reply by maboloshi** [reply]
就是说 github又更新了是吧,如果你那边我和一样的话 我来调整一下吧
另外 这个链接 https://github.com/github-copilot/signup 也没了吧

**Reply by TC999** [reply]
> 就是说 github又更新了是吧

对，还新增了选项

**Reply by TC999** [reply]
> 就是说 github又更新了是吧,如果你那边我和一样的话 我来调整一下吧 另外 这个链接 https://github.com/github-copilot/signup 也没了吧

还在

**Reply by maboloshi** [reply]
这个 https://github.com/settings/copilot 我来更新一下, 顺便增加 https://github.com/github-copilot/free_signup

**Reply by TC999** [reply]
> 这个 https://github.com/settings/copilot 我来更新一下, 顺便增加 https://github.com/github-copilot/free_signup

刚添加了部分词条 #322

**Comment by Paul-16098** [comment]
https://github.com/settings/personal-access-tokens

![image](https://github.com/user-attachments/assets/4cb9f8d3-d0ac-4646-89bd-7730ca9084f6)

**Comment by LuYifei2011** [comment]
[https://github.com/scratchblocks/scratchblocks/pull/556/checks?sha=b4d8ce6fb3e696268fc296a1488d4e6c15961842](https://github.com/scratchblocks/scratchblocks/pull/556/checks?sha=b4d8ce6fb3e696268fc296a1488d4e6c15961842)

<img width="2880" height="1704" alt="image" src="https://github.com/user-attachments/assets/8ec86667-46c3-4917-80c3-cb95b40ac94d" />

**Comment by wyc-26** [comment]
追加一处：https://github.com/pulls
注：需先打开功能预览中的新版拉取请求仪表板

<img width="2520" height="1342" alt="image" src="https://github.com/user-attachments/assets/15d240af-f1c5-47fc-a6d0-c5d24724f771" />
